### PR TITLE
chore(site): Add Vercel logo to footer

### PR DESCRIPTION
**Pull request by montogeek**
Vercel has given us a Pro account that we have used since ~3 weeks ago, now we add a small banner to our footer that link back to them, it is also added to the README file.

### CODE REVIEW

**Comment by EugeneHlushko** [comment]
Still wondering if we should do the footer link or just readme link (since actual hosting for prod is GH pages) @sokra wdyt?

**Comment by montogeek** [comment]
Yeah, not sure, easier to remove code than add it :)

**Comment by sokra** [comment]
I think we should not falsy claim that it's powered by vercel, for the pages that are really powered by gh-pages. This might gives negative PR to vercel, e. g. in causes when GitHub performs worse.

So only add that logo when it's true.

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import Container from '../Container/Container';
 import Icon from '../../assets/icon-square-small.svg';
 import CC from '../../assets/cc.svg';
 import BY from '../../assets/by.svg';
+import Vercel from '../../assets/powered-by-vercel.svg';
```
Comment (EugeneHlushko) [inline src/components/Footer/Footer.jsx:7]:
```suggestion
```

File: src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -36,6 +37,9 @@ const Footer = () => (
             alt="Creative Commons License"
             src={ BY } />
         </Link>
+        <Link className="footer__link footer__license" to="https://vercel.com/?utm_source=webpackdocs">
+          <img src={ Vercel } alt="Powered by Vercel" />
+        </Link>
```
Comment (EugeneHlushko) [inline src/components/Footer/Footer.jsx:42]:
```suggestion
```

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**